src/services/anchor-service.ts

```diff
@@ -53,10 +53,7 @@ class IpfsMerge implements MergeFunction<Candidate> {
   }
 
   async merge(left: Node<Candidate>, right: Node<Candidate>): Promise<Node<Candidate>> {
-    const merged = {
-      L: left.data.cid,
-      R: right.data.cid
-    };
+    const merged = [left.data.cid, right.data.cid];
```

Comment:
It bothers me that I can change this line without changing the corresponding line below for how we generate the `path` to put into the anchor record, and no tests break.  That's because the `path` is generated here but only consumed in js-ceramic, and we don't have good integration tests between js-ceramic and real CAS nodes yet.  I filed https://github.com/ceramicnetwork/ceramic-anchor-service/issues/199 to fix that.

Reply:
Thanks for creating that ticket.